code/obj/item/misc_weapons.dm

```diff
@@ -579,6 +584,7 @@ TYPEINFO(/obj/item/sword/pink/angel)
 /obj/item/dagger/syndicate
 	name = "syndicate dagger"
 	desc = "An ornamental dagger for syndicate higher-ups. It sounds fancy, but it's basically the munitions company equivalent of those glass cubes with the company logo frosted on."
+	pierces_armor = TRUE
 	HELP_MESSAGE_OVERRIDE({"Throw the dagger at someone to instantly incapacitate them for a short while."})
 
 /obj/item/dagger/syndicate/specialist //Infiltrator class knife
```

Comment:
feels like it would make more sense to split the inheritance here, since the specialist dagger doesn't inherit anything other than the help message, which ought to be on the parent anyways

Reply:
Sure